modules/ref_runtimes_metering_labels.adoc

```diff
@@ -0,0 +1,17 @@
+//Include this reference module in product release notes.
+//Be sure you declare runtimes-attributes.doc
+//Content is intended for CADS documentation teams.
+
+[id='runtimes_metering_labels-{context}']
+= {ProductName} metering labels for Red Hat OpenShift
+
+The following metering labels apply to {ProductName} pods on OpenShift:
```

Comment:
{ProductName} clusters on {openshiftshort} can use the following metering labels to check compliance.

Reply:
Thanks, Sangeeta. This sounds like an unfinished statement to me though. Compliance with what?